CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+# 4.0.0 (2020-05-14)
+
+### New features
+
+- Added a test fixture for the list view checklist component to be used in consumer unit tests. [#45](https://github.com/blackbaud/skyux-list-builder-view-checklist/pull/45)
```

Comment:
Swap "to be used" for "to use" here and in line 16, and you're good to go!

Reply:
Done!